docusaurus.config.js

```diff
@@ -109,6 +115,11 @@ const config = {
                 to: "https://www.palisadoes.org/contact/",
                 className: "footer__icon footer__contact",
               },
+              {
+                label: ' FAQ',
+                to: '/docs/faq-section',
```

Comment:
Why do we need this? There will be a FAQ in the header

Reply:
It seamed reasonable to me to add the link for FAQ both in header and footer, as it's the most important section for the users to catch their attention
but if it's not required and suggested by mentors/maintainers I'm ready to remove it. Thanks!